perceval/backends/core/confluence.py

```diff
@@ -91,6 +92,16 @@ def fetch(self, category=CATEGORY_HISTORICAL_CONTENT, from_date=DEFAULT_DATETIME
 
         return items
 
+    @staticmethod
```

Comment:
Why static?

Reply:
No particular reason, other than I didn't need anything from `self` object. If you prefer to have it non-static I can change it.